Source/Renderer/VertexArray.js

```diff
@@ -275,17 +290,29 @@ define([
             }
         }
 
+        //>>includeStart('debug', pragmas.debug);
         // Verify all attribute names are unique
         var uniqueIndices = {};
-        for ( var j = 0; j < vaAttributes.length; ++j) {
-            var index = vaAttributes[j].index;
+        for (i = 0; i < vaAttributes.length; ++i) {
+            var index = vaAttributes[i].index;
             if (uniqueIndices[index]) {
                 throw new DeveloperError('Index ' + index + ' is used by more than one attribute.');
             }
-
             uniqueIndices[index] = true;
         }
 
+        // Verify that not all attributes are instanced
+        if (vaAttributes.length > 0) {
+            var allAttributesInstanced = true;
```

Comment:
Performance doesn't really matter here, but perhaps:
```javascript
var allAttributesInstanced = true;
var length = vaAttributes.length;
for (i = 0; i < length; ++i) {
    if (vaAttributes[i].instanceDivisor === 0) {
      allAttributesInstanced = false;
      break;
    }
}
```
This should almost always break at the first attribute.